translate/translate.js

```diff
@@ -132,28 +123,28 @@ var program = module.exports = {
 
 cli
   .demand(1)
-  .command('detect <text>', 'Detect the language of the provided text', {}, function (options) {
-    program.detectLanguage(options.text, options.apiKey, utils.makeHandler(false));
+  .command('detect <input..>', 'Detect the language of the provided text or texts', {}, function (options) {
+    process.env.TRANSLATE_API_KEY || (process.env.TRANSLATE_API_KEY = options.apiKey);
```

Comment:
I like the brevity - but I'm wondering whether or not this be clearer as an `if` statement e.g.
```javascript
if (!process.env.TRANSLATE_API_KEY) {
    process.env.TRANSLATE_API_KEY = options.apiKey;
}